apps/dashboard/app/models/project.rb

```diff
@@ -306,17 +303,17 @@ def adapter(cluster_id)
   end
 
   def readme_path
-    file = Dir.glob("#{directory}/README.{md,txt}").first.to_s
+    file = Dir.glob("#{directory.to_s}/README.{md,txt}").first.to_s
```

Comment:
The linter should tell you that you don't need to cast to_s if you're string interpolating through `#{}`.